cmd/rancher_release/check_rancher_rc_deps.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"errors"
+
+	"github.com/rancher/ecm-distro-tools/release/rancher"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+func checkRancherRCDepsCommand() *cli.Command {
+	return &cli.Command{
+		Name:  "check-rancher-rc-deps",
+		Usage: "check if the Rancher version specified by the commit or pre-release title does not contain development dependencies and rc tags",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:     "commit",
+				Aliases:  []string{"c"},
+				Usage:    "last commit for a final rc",
+				Required: false,
+			},
+			&cli.StringFlag{
+				Name:     "release-title",
+				Aliases:  []string{"rt"},
+				Usage:    "release title from a given release process",
+				Required: false,
+			},
+			&cli.StringFlag{
```

Comment:
I'm not sure if this is necessary since the code base is capable of deriving the org from the repo name.